dev/gp-gcloud/cmd/compute/instance-templates-create.go

```diff
@@ -0,0 +1,300 @@
+	var (
+		region               string
+		project              string
+		machineType          string
+		minCpuPlatform       string
+		serviceAccount       string
+		serviceAccountScopes string
+		bootDiskSize         string
+		bootDiskType         string
+		tags                 string
+		labels               string
+		image                string
+		networkInterface     string
+		metadata             string
+		metadataFromFile     string
+		localSSDs            []string
+		noRestartOnFailure   bool
+		dryRun               bool
+	)
+
+	cmd := &cobra.Command{
+		Use:     "create",
+		Short:   "create wrapper",
+		Example: "  create instance-template-1",
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) != 1 {
+				log.Fatal("no name was provided")
+			}
+
+			name := args[0]
+
+			ctx := context.Background()
+
+			c, err := google.DefaultClient(ctx, compute.CloudPlatformScope)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			computeService, err := compute.New(c)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			networkInterfaceParsed := parseLabels(networkInterface)
+			automaticRestart := !noRestartOnFailure
+			metadata := parseMetadata(metadata)
+			metadata = append(metadata, parseMetadataFromFile(metadataFromFile)...)
+			localSSDDevices := parseLocalSSDs(localSSDs)
+			labels := parseLabels(labels)
+
+			rb := &compute.InstanceTemplate{
+				Name: name,
+				Properties: &compute.InstanceProperties{
+					MachineType:    machineType,
+					MinCpuPlatform: minCpuPlatform,
+					ServiceAccounts: []*compute.ServiceAccount{
+						{
+							Email:  serviceAccount,
+							Scopes: strings.Split(serviceAccountScopes, ","),
+						},
+					},
+					Disks: []*compute.AttachedDisk{
+						{
+							AutoDelete: true,
+							Boot:       true,
+							InitializeParams: &compute.AttachedDiskInitializeParams{
+								DiskSizeGb:  parseDiskSize(bootDiskSize, 10),
+								DiskType:    bootDiskType,
+								SourceImage: image,
+								Labels:      labels,
+							},
+						},
+					},
+					Tags: &compute.Tags{
+						Items: strings.Split(tags, ","),
+					},
+					Labels: labels,
+					NetworkInterfaces: []*compute.NetworkInterface{
+						{
+							AccessConfigs: []*compute.AccessConfig{
+								{
+									NetworkTier: networkInterfaceParsed["network-tier"],
+									Type:        "ONE_TO_ONE_NAT",
+								},
+							},
+							Network:    networkInterfaceParsed["network"],
+							Subnetwork: networkInterfaceParsed["subnet"],
+						},
+					},
+					Metadata: &compute.Metadata{
+						Items: metadata,
+					},
+					Scheduling: &compute.Scheduling{
+						AutomaticRestart: &automaticRestart,
+					},
+				},
+			}
+
+			for _, localSSD := range localSSDDevices {
+				rb.Properties.Disks = append(rb.Properties.Disks, &compute.AttachedDisk{
+					AutoDelete: true,
+					Boot:       false,
+					InitializeParams: &compute.AttachedDiskInitializeParams{
+						DiskType: "local-ssd",
+						Labels:   labels,
```

Comment:
This is the label set we cannot set w/o the CLI, right?

Can we test this w/o building an ephemeral cluster? For example, after building this template, can we curl it to determine what these properties should look like?

Reply:
you can check those labels by inspecting created instance template in gcp.

Reply:
I did not see them in my instance template...can you share a screenie of what you mean given a similar example created with this CLI?

Reply:
![image](https://user-images.githubusercontent.com/18602811/178037001-a77e69b4-223d-4de5-a4af-28c4c05fa0bf.png)

Reply:
TIL about `equivalent rest response`, thanks @sagor999 ! :handshake: 

I was expecting to see labels on both the boot and local SSD disks, but, only see them on boot. :thinking: Am I missing something?

Reply:
Updated code to reflect that.

Reply:
Roger that! I just wrapped an IDE incident, and it's time to get ice cream with the Mrs., so I"ll pick this up on Monday. :wave: :ice_cream: :spoon: 

Thank you for the update, bud, have a nice weekend! :tada: